app/Actions/Database/StopDatabase.php

```
@@ -18,23 +18,21 @@ class StopDatabase
 {
     use AsAction;
 
-    public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|StandaloneMysql|StandaloneMariadb|StandaloneKeydb|StandaloneDragonfly|StandaloneClickhouse $database, bool $dockerCleanup = true)
+    public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|StandaloneMysql|StandaloneMariadb|StandaloneKeydb|StandaloneDragonfly|StandaloneClickhouse $database, bool $dockerCleanup = true, bool $resetRestartCount = true, bool $removeContainer = true)
```

PHPMD (2.15.0)

[error] The method handle has a boolean flag argument $dockerCleanup, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)